Pages/Index.cshtml.cs

```diff
@@ -68,7 +68,7 @@ public void OnGet(string jsonSting)
                    
                   
 
-                    var propertyResidential = Property.FromJson(propertyResJason);
+                    var propertyResidential = Property.FromJson(ResidentialPropertyType);
```

Comment:
Renamed the variable which describes Residential Property Types. Reduces technical debt by improving the readability quotient of the code. Also, the necessity of inline comments is reduced.

Sources
https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/inside-a-program/coding-conventions